### ISSUE TITLE
[Accessibility]A11y__AzureDataStudio_Proxiestab_Screenreader: NVDA  is not announcing the table name

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#5612]
[Check out Keros!](https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
 GitHubTags: #A11yMAS; #A11yTCS; #MAS1.3.1;#MAS4.1.2 #A11y_AzureDataStudio; #Win32; #DesktopApp
**Environment Details:**
Application name: Azure Data Studio 
Version: 1.8.0-insider (user setup)
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows10

**Additional Details:**
Screen reader name: NVDA
NVDA Version: 2019.1
JAWS version:2019
MAS reference: MAS1.3.1,MAS4.1.2.

**Repro Steps:**
1. Install the app Azure Data Studio.
2. Connect to Database-> Go to the  SQL server agent tab->Select proxies tab
 3. Start NVDA.
4. Navigate to table.
5. Observe that screen reader doesn’t announce the table name .

**Actual:**
While navigating through table, screen reader doesn’t announce the table name.

**Same issue observed with JAWS.**
**Similar issue is observed for the below pages:**
 **1.Jobs:** Navigate to the connected server->Select the database->Navigate to the SQL agent tab->
Navigate to jobs page.
**2.Operatortab:** Navigate to the connected server->Select the database->Navigate to the SQL agent tab->Navigate to Operator tab
**3. Alertstab:** Navigate to the connected server->Select the database->Navigate to the SQL agent tab->Navigate to Alerts tab.
**4.CreateAlertResponse:** Go to the  SQL server agent tab->On alerts tab->Click on new alert button->Select response tab.
**5.NewJobSchedule:** Navigate to the SQL agent tab from Home page->Click on new job button->Select schedule tab.
**6. NewJobAlerts:** Navigate to the connected server->Select the database->Navigate to the SQL agent tab->Click on New Job button-> Navigate to Alerts tab.     
7.CSVModifiedColumns:
8.**CSVPreviewData**-Connect to database->navigate to the Data services folder->HDFS Folder-> navigate to CSV files folder->Right click on the files->Select option Create external table from CSV->Click on the "Next" button.
9.**CSVSummar**y- Connect to database->navigate to the Data services folder->HDFS Folder-> navigate to CSV files folder->Right click on the files->Select option Create external table from CSV->Click on the "Next" button->click on next button.
**10.CSVModifyColumn:** Connect to database->navigate to the Data services folder->HDFS Folder-> navigate to CSV files folder->Right click on the files->Select option Create external table from CSV->Click on the "Next" button->Click on next button.
10.**Summary**- Enable SQL server2019 preview->select table from the database->Create External Table screen should open->Next button->Create Database Master Key screen should open->click on next button->fill all the required details->click on next button->Summary screen should open->Click on next button.

**Expected:**
Screen reader should announce the table name.

**User Impact:**
If screen reader user wants to navigate through only table it won't understand which table they are navigating.

**Recommendation:**
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

**MAS Reference:**
**MAS4.1.2-**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}

**MAS1.3.1**-https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7B54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4%7D&cid=1a1d7dfd-3dfd-4394-a8af-81fa7c1cf482 
**Attachment for Reference:**
[5612_[Accessibility]A11y__Azure Data Studio_Proxiestab_Screenreader.pptx](https://github.com/microsoft/azuredatastudio/files/3215609/5612_.Accessibility.A11y__Azure.Data.Studio_Proxiestab_Screenreader.pptx)

Does this issue occur when all extensions are disabled?: Yes
